src/CalcViewModel/Common/DateCalculator.cpp

```diff
@@ -12,6 +12,7 @@ using namespace CalculatorApp::Common::DateCalculation;
 DateCalculationEngine::DateCalculationEngine(_In_ String^ calendarIdentifier)
 {
     m_calendar = ref new Calendar();
+    m_calendar->ChangeTimeZone("UTC");
```

Comment:
Worthwhile to add a comment into here as to why.  This has clearly been a question that has come up in the code in the past.